frontend/src/app/components/ExperimentPage.js

```diff
@@ -34,7 +35,8 @@ export default class ExperimentPage extends React.Component {
       progressButtonWidth: null,
       showDisableDialog: false,
       shouldShowTourDialog: false,
-      showTourDialog: false
+      showTourDialog: false,
+      showPreFeedbackDialog: false
```

Comment:
Let's do all of this in #1367, since we're going to be adding all of this stuff to the redux store there anyway.